content/3_Silver/Prefix_Sums_2.mdx

```diff
@@ -172,6 +172,47 @@ int main() {
 ```
 
 </CPPSection>
+	
+<JavaSection>
+	
+### Java Implementation
+
+```java
+import java.io.*;
+import java.util.*;
+public class maxSubarraySumKadane {
```

Comment:
Add whitespace between lines 182 and 183, and change the name of the class so it's PascalCase please.